docs/versioned_docs/version-2.10.0/widgets/icon.md

```diff
@@ -73,5 +73,5 @@ Following actions of the component can be controlled using the component specifi
 
 | Actions     | Description |
```

Comment:
Hey @niranjan-kurhade, here you need to add the "How To Access" column and then update the content accordingly.